buildchain/buildchain/config.py

```diff
@@ -47,11 +47,18 @@
 
 # Name of the command (if in the PATH) or path to the binary.
 
-DOCKER   : str = os.getenv('DOCKER_BIN',   'docker')
+# DOCKER   : str = os.getenv('DOCKER_BIN',   'docker')
```

Comment:
Following @gdemonet recommendation, we will give up the `dict`-based approach and move onto `enum`.

This will simplify the code (we no longer need to use `CmdAction`).

This should become:

```python
# External commands {{{

# Name of the external commands (if in the PATH) or path to the binary.

class ExtCommand(enum.Enum):
    """External commands used by the build chain."""

    GIT      = os.getenv('GIT_BIN',      'git')
    HARDLINK = os.getenv('HARDLINK_BIN', 'hardlink')
    MKISOFS  = os.getenv('MKISOFS_BIN',  'mkisofs')
    SKOPEO   = os.getenv('SKOPEO_BIN',   'skopeo')
    VAGRANT  = os.getenv('VAGRANT_BIN',  'vagrant')

# }}}
```

Reply:
I think that the GIT_BIN should not be included here. no??